### DISCUSSION TITLE
Implementing Toast with Color Customization

### DISCUSSION THREAD

**Discussion by gustavosobrinho01** [livewire/flux#159] [Feature Requests] [answered]
I would like to start a discussion about implementing color customization for Toast notifications in our project. Currently, Toasts use a standard design without much flexibility in terms of color. My suggestion is to allow developers to specify custom colors (e.g., success, warning, error) to enhance the user experience and provide more meaningful feedback.

What are your thoughts on this proposal? How can we best implement color customization in a way that is flexible but also easy to maintain?

**Comment by cieplik206** [comment]
Yes, publishing toasts assets is not a problem solver becouse you cannot pass "type" of toasts using Flux::toast() from livewire.

**Comment by JoshMountain** [comment]
I believe this is a duplicate of https://github.com/livewire/flux/discussions/132

**Comment by dudelisius** [comment, marked as answer]
20 hours ago the latest version 1.0.9 was release and this supports variants; https://fluxui.dev/components/toast#variants
